src/main/java/com/google/cloud/spanner/jdbc/JdbcStatement.java

```diff
@@ -401,15 +400,6 @@ void checkAndSetBatchType(String sql) throws SQLException {
     }
   }
 
-  private void checkConnectionHasNoActiveBatch() throws SQLException {
```

Comment:
This check is no longer needed, as the Java Connection API now allows us to 'execute a batch in a batch'. The statements that are executed in this batch, are just added to the existing batch on the connection.